src/Tizen.NUI/src/public/Utility/Palette.cs

```diff
@@ -204,7 +203,6 @@ public static Palette Generate(PixelBuffer pixelBuffer, Rectangle region)
         /// Returns all of the swatches which make up the palette.
         /// </summary>
         /// <returns>The swatch list</returns>
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public ReadOnlyCollection<Swatch> GetSwatches()
```

Comment:
it's recommended to declare returning interfaces rather than specific classes, Please add one letter to signature and keep the implementation as it is
```suggestion
        public IReadOnlyCollection<Swatch> GetSwatches()
```